plugins/node/opentelemetry-instrumentation-express/examples/package.json

```diff
@@ -26,24 +27,25 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "bugs": {
-    "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
+    "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
   },
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-express/examples#readme",
   "dependencies": {
-    "@opentelemetry/api": "^1.0.0",
-    "@opentelemetry/exporter-jaeger": "^1.0.0",
-    "@opentelemetry/exporter-zipkin": "^1.0.0",
+    "@opentelemetry/api": "^1.3.0",
+    "@opentelemetry/exporter-jaeger": "^1.18.1",
+    "@opentelemetry/exporter-trace-otlp-proto": "^0.46.0",
+    "@opentelemetry/exporter-zipkin": "^1.18.1",
     "@opentelemetry/instrumentation": "^0.46.0",
-    "@opentelemetry/instrumentation-express": "0.28.0",
+    "@opentelemetry/instrumentation-express": "^0.33.3",
     "@opentelemetry/instrumentation-http": "^0.46.0",
-    "@opentelemetry/resources": "^1.0.0",
-    "@opentelemetry/sdk-trace-base": "^1.0.0",
-    "@opentelemetry/sdk-trace-node": "^1.0.0",
-    "@opentelemetry/semantic-conventions": "^1.0.0",
+    "@opentelemetry/resources": "^1.18.1",
+    "@opentelemetry/sdk-trace-base": "^1.18.1",
+    "@opentelemetry/sdk-trace-node": "^1.18.1",
+    "@opentelemetry/semantic-conventions": "^1.18.1",
```

Comment:
Now these need another update to ^1.20.0 etc, right?

Then I'm not sure if the various examples/package.json files will be included in the deps updates processes.

Reply:
I was wrong. Those caret version ranges are fine. (They could probably be reduced to lower versions, e.g. `"@opentelemetry/resources": "^1.0.0",`, but I don't think that is required for anything.)

The important thing is that the versions in package-lock.json are good.